include/Metabox/Base.php

```diff
@@ -301,4 +301,41 @@ protected function render_result( $posts_moved ) {
 			'updated'
 		);
 	}
+
+	protected function render_roles_dropdown( $name = '', $hide_no_role_option = false ) {
+		global $wp_roles;
+		$roles = $wp_roles->roles;
+
+		if ( ! ( $users = wp_cache_get( 'bm_users', 'bulk-wp' ) ) ) {
+			$users = get_users();
+			wp_cache_add( 'bm_users', 'bulk-wp' );
+		}
+
+		$bulk_move = bulk_move();
+
+		$users_by_roles = $bulk_move->helper->get_users_count_by_roles( $users, $roles );
```

Comment:
This function will be part of `UserMetabox` and let's remove the dependency to `bulk_move()`.

Reply:
> This function will be part of UserMetabox and let's remove the dependency to bulk_move().

Fixed.